packages/env/lib/commands/install-path.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' );
+
+/**
+ * Internal dependencies
+ */
+const { readConfig } = require( '../config' );
+
+/**
+ * Logs the path to where wp-env files are installed.
+ */
+module.exports = async function installPath() {
+	const configPath = path.resolve( '.wp-env.json' );
+	if ( ! fs.existsSync( configPath ) ) {
```

Comment:
Sorry for going back and forth on this so much! It is possible to run `wp-env` without the JSON configuration file (iirc), so I might remove this portion of the check. Otherwise this looks great :+1:

Reply:
Ah, slipped my mind! You're right.

Reply:
I will remove it today or tomorrow... Or you can do it if you feel ilke it :)